src/badguy/badguy.cpp

```diff
@@ -1064,7 +1064,7 @@ BadGuy::after_editor_set()
 {
   MovingSprite::after_editor_set();
 
-  if (m_dir == Direction::AUTO)
+  if (m_start_dir == Direction::AUTO)
   {
     if (m_sprite->has_action("editor-left")) {
```

Comment:
Could this monstrosity of an if/else statement not be converted to a list and a foreach?

Reply:
Lmfao